### ISSUE TITLE
Autocomplete missing required aria-roles

### ISSUE THREAD

**Issue by arcepaul** [NCIOCPL/ncids#663]
It was noticed in the lighthouse report that the autocomplete in the header raises the following issue, lowering the accessibility score for the page: 
`[role]s do not have all required [aria-*] attributes`

> **ESTIMATE** TBD

Figure out what is missing to remediate this error

Resources: 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-autocomplete

**Comment by arcepaul** [comment]
It looks like autocomplete needs to reflect the state of the listbox it owns via `aria-expanded`
